Source/WebKit/UIProcess/WebProcessPool.cpp

```diff
@@ -818,9 +818,10 @@ void WebProcessPool::registerHighDynamicRangeChangeCallback()
 
 WebProcessDataStoreParameters WebProcessPool::webProcessDataStoreParameters(WebProcessProxy& process, WebsiteDataStore& websiteDataStore)
 {
-    websiteDataStore.resolveDirectoriesIfNecessary();
+    websiteDataStore.waitDirectoriesToResolveIfNecessary();
```

Comment:
I would rename this to `waitForDirectoriesToResolveIfNecessary()`. It sounds a little weird otherwise.